MAINTAINERS_RULES.md

```diff
@@ -0,0 +1,39 @@
+# Maintainers Rules
+
+This document lays out some basic rules and guidelines all maintainers are expected to follow.
+Changes to the acceptance criteria for merging PRs require a ceiling(two-thirds) supermajority from the maintainers.
+Changes to the Guidelines require a simple majority.
```

Comment:
nit: is this meant to refer to repo guidelines?  Might be helpful to have hyperlinks to the sections for this and acceptance criteria above to be explicit.